src/test/regress/expected/multi_repair_shards.out

```diff
@@ -1,4 +1,6 @@
 ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 820000;
+SELECT groupid AS worker_2_group FROM pg_dist_node WHERE nodeport=:worker_2_port \gset
+SELECT groupid AS worker_1_group FROM pg_dist_node WHERE nodeport=:worker_1_port \gset
```

Comment:
`:worker_1_group` might work but the second worker's group changes a ton, tests keep removing and adding the second node and it gets a new group every time.